account_ebics_batch/data/ir_cron_data.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo noupdate="1">
+
+  <record id="ir_cron_ebics_batch_import" model="ir.cron">
+    <field name="name">EBICS Batch Import</field>
+    <field name="model_id" ref="model_ebics_batch_log" />
+    <field name="state">code</field>
+    <field name="code">model._batch_import()</field>
```

Comment:
Is it maybe specific to our bank but downloading without `date_from` and `date_to` results in no transferred files.

Would it hurt to only download the files from yesterday in this daily cron?

```
today = datetime.date.today() - datetime.timedelta(days=1)
model._batch_import()(config, yesterday, yesterday)
```

Reply:
Alternatively, download from date since last download. But what to do if there is no previous download?

Reply:
You can always do a manual import for the time before. It could also be the case that the bank blocks the download because there are too many files.

Otherwise it would be:
```
yesterday = datetime.date.today() - datetime.timedelta(days=1)
# Get the dates from the previous successful runs
domain = [("state", "=", "done"), ("date_to", "!=", False)]
previous = env["ebics.batch.log"].search(domain).mapped("date_to")
start = max(start) if start else yesterday
model._batch_import(date_from=start, date_to=yesterday)
```

Reply:
@fkantelberg @nikunjantala who of you could fix it?

Reply:
@jans23 @fkantelberg 
I haven't yet seen a bank where you should specify the date range on the download.
Maybe you are testing with the 'free' version of the fintech license.
If so, please order a license key (cf. https://www.joonis.de/en/fintech/) since the free licence is unfortunately not suited for automated operations.

Reply:
Actually happend the second time on a different german bank with a paid license but I can understand it that you don't want to add such a snippet when it works for other banks.